### DISCUSSION TITLE
Newsletter

### DISCUSSION THREAD

**Discussion by thunderclap82** [seerr-team/seerr#166] [Ideas]
I would love an option for a Newsletter like Tautulli and Ombi have, that at a scheduled time an email blast goes out to registered users letting them know all the new content added since the previous week.

**Comment by fallenbagel** [comment]
> I would love an option for a Newsletter like Tautulli and Ombi have, that at a scheduled time an email blast goes out to registered users letting them know all the new content added since the previous week.

Better add this as a feature request in the issues tab c:

**Reply by thunderclap82** [reply]
Done. Thank you.
